docs/guides/preflight-checklist-few-shot.md

```
@@ -212,16 +212,18 @@ If only some chunks are mismatched, retrieval quality degrades. Always validate
 
-  If missing, generate (takes ~65 min for 58 participants):
-  ```bash
-  # Generate HuggingFace FP16 embeddings (recommended)
-  uv run python scripts/generate_embeddings.py --split paper-train
-  # Output: data/embeddings/huggingface_qwen3_8b_paper_train.npz
-
-  # Or generate Ollama embeddings (paper-parity)
-  EMBEDDING_BACKEND=ollama uv run python scripts/generate_embeddings.py --split paper-train
-  # Output: data/embeddings/ollama_qwen3_8b_paper_train.npz
-  ```
+	  If missing, generate (takes ~65 min for 58 participants):
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)